src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpInstructionDecoder.cs

```diff
@@ -131,7 +131,7 @@ internal override CSharpCompilation GetCompilation(DkmClrInstructionAddress inst
             var moduleInstance = instructionAddress.ModuleInstance;
             var appDomain = moduleInstance.AppDomain;
             var previous = appDomain.GetDataItem<CSharpMetadataContext>();
-            var metadataBlocks = instructionAddress.Process.GetMetadataBlocks(appDomain);
```

Comment:
Can we just change this method to take a DkmClrModuleInstance (instead of the DkmClrInstructionAddress)?